railgun/test/integration/ingress_test.go

```diff
@@ -147,7 +149,9 @@ func TestIngressEssentials(t *testing.T) {
 			// now that the ingress backend is routable, make sure the contents we're getting back are what we expect
 			// Expected: "<title>httpbin.org</title>"
 			b := new(bytes.Buffer)
-			b.ReadFrom(resp.Body)
+			n, err := b.ReadFrom(resp.Body)
+			require.NoError(t, err)
+			require.True(t, n > 0)
```

Comment:
Question:
  if we can check specific keyword, would make more sense comparing to require.True(t, n > 0). Possibly, content w/o expected data. 
  What you think about it ?

Reply:
Given that we test the actual contents of the body below, we would never make it pass this test if the contents weren't consistent with what we expect, so I don't see any particular value in checking the exact content length beyond making lint happy. I would suggest we can probably leave this OR if we feel strongly we could set the check against the minimum number of bytes needed for valid HTML (which I believe is 13) but I'm really not sure what that gains us?

Reply:
Understand your points. 

Here is an example of how we gain quality:
 
Adding following lint after the line
require.True(t, strings.Contains(bodyString, "Hello Go Sample v1!"))
https://github.com/Kong/kubernetes-ingress-controller/pull/1537/files#diff-3a5e32729a670957ded339e81eff75042add412c04794c58b2226c0930c95963L221

I have  been assuming checking content is the purpose of caring about resp.Body on top of resp.HttpStatus (200).

This is not a blocker for checkin, but justifying adding the change.

Reply:
The line below checks the body string, so I think we're all set here, let me know if there's some other improvement you're looking for, or if I've misunderstood the intention here because we can always improve these tests in follow up iterations too.